website/azurerm.erb

```diff
@@ -788,9 +820,17 @@
                   <a href="/docs/providers/azurerm/r/virtual_machine_extension.html">azurerm_virtual_machine_extension</a>
                 </li>
 
+                <li>
+                  <a href="/docs/providers/azurerm/r/linux_virtual_machine_scale_set.html">azurerm_linux_virtual_machine_scale_set</a>
+                </li>
+
                 <li>
                   <a href="/docs/providers/azurerm/r/virtual_machine_scale_set.html">azurerm_virtual_machine_scale_set</a>
                 </li>
+
+                <li>
+                  <a href="/docs/providers/azurerm/r/windows_virtual_machine_scale_set.html">azurerm_windows_virtual_machine_scale_set</a>
+                </li>
```

Comment:
this hasn't launched yet (it's feature toggled off) - so can we remove this for the moment?